app/actors/curation_concerns/actors/apply_order_actor.rb

```diff
@@ -3,11 +3,28 @@ module Actors
     class ApplyOrderActor < AbstractActor
       def update(attributes)
         ordered_member_ids = attributes.delete(:ordered_member_ids)
+        sync_members(ordered_member_ids)
         apply_order(ordered_member_ids) && next_actor.update(attributes)
       end
 
       private
 
+        def sync_members(ordered_member_ids)
```

Comment:
Either works - an actor spec is probably better, if you can swing it. But https://coveralls.io/builds/6714170/source?filename=app%2Factors%2Fcuration_concerns%2Factors%2Fapply_order_actor.rb#L17 it's not covered.